app/controllers/home_controller.rb

```diff
@@ -1,4 +1,12 @@
-class HomeController < WebsiteController
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
```

Comment:
HomeController has at least 5 instance variables

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Too-Many-Instance-Variables.md)